### DISCUSSION TITLE
How to change the pages direction from rtl to ltr?

### DISCUSSION THREAD

**Discussion by MohAid** [zauberzeug/nicegui#2266] [Q&A] [answered]
### Question

How can I change **_at the runtime_** the direction of the pages from ltr to rtl and vice versa?
Let's say I will bind that from `'lang_dir'` variable.
I can solve the binding issue, I just need to know how to reach and change it.
And do I have to change something else in this case?

note: I changed this manually and I got what I needed:
![image](https://github.com/zauberzeug/nicegui/assets/78490169/e6bb6040-0c0e-48cb-a243-314a61c30f4a)

And by the way how to change elements classes?
I tried to do this for example:
```
lbl = ui.label().classes('p-2 items-left')
lbl.classes('items-right')
```
The result as appeared on the console was 'p-2 items-left items-right' and also without any effect!

**Comment by rodja** [comment, marked as answer]
You can reach the body tag with [`ui.query('body')`](https://nicegui.io/documentation/query#set_background_gradient) and remove classes with [`lbl.classes('items-right', remove='items-left')`](https://nicegui.io/documentation/section_styling_appearance#try_styling_nicegui_elements_:~:text=All%20three%20functions%20also%20provide%20remove%20and%20replace%20parameters).

**Reply by MohAid** [reply]
Thanks @rodja 🥇 
I used it like this:
`ui.query('body').style(f'direction: "rtl"]}')`

The problem was I used `.classes` instead of `.style`!
The other problem was that I was using it inside a binded function so it was not recognizing it, I defined it outside before the function `body = ui.query('body')` then I changed the direction like this `body.style(f'direction: "rtl"]}')`
